index.js

```diff
@@ -14,15 +16,21 @@ var anymatch = function(criteria, value, returnIndex, startIndex, endIndex) {
   }
   startIndex = startIndex || 0;
   var string = value[0];
+  var altString;
+  if (platform === 'win32' && typeof string === 'string') {
```

Comment:
I need it to be something I can override in the tests.

Is `path.sep` ever not backslash on windows, and is it ever backslash on any other platform?